src/test/java/com/hocs/test/pages/workstacks/Workstacks.java

```diff
@@ -1,10 +1,25 @@
 package com.hocs.test.pages.workstacks;
 
+import static org.junit.Assert.fail;
+
+import java.util.ArrayList;
+import java.util.List;
 import net.serenitybdd.core.annotations.findby.FindBy;
 import net.serenitybdd.core.pages.WebElementFacade;
 import com.hocs.test.pages.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+public class Workstacks extends Page{
+
+    @FindBy(xpath = "//td[@class=\"govuk-table__cell\" and text()=\"danny.large@ten10.com\"]")
```

Comment:
you can use single quotes for things like this to remove the \ ie 
"//td[@class='govuk-table__cell' and text()='danny.large@ten10.com']"